tests/storage/blockvolume_test.py

```diff
@@ -160,7 +162,7 @@ def test_optimal_size_cow_leaf_max(self):
         (512 * MiB, 200 * MiB, 640 * MiB),
         # Add minimum padding.
         (2 * GiB, 1023 * MiB, 1024 * MiB),
-        (2 * GiB, 1024 * MiB, 1024 * MiB + blockVolume.MIN_PADDING),
+        (2 * GiB, 1024 * MiB, 1152 * MiB),
```

Comment:
I'm not a big fan of inline comments, but here maybe helps to have an explanation of what is tested with the optimal size of this permutation?

Reply:
Yes, we need a comment for each case.